### PR TITLE
Created blog 

### PR DESCRIPTION
**Pull request by brad-j**
I've created the blog list page and the blog posts page.  

The styling on the individual posts could use some love. 

My first ever PR on Your First PR, so meta.

### CODE REVIEW

**Comment by brad-j** [comment]
There's a mistake in header.html, line 7. It should be coc.html. (It's ./coc.html in the PR)

File: _layouts/blog-post.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+{% include head.html %}
```
Comment (nickcolley) [inline _layouts/blog-post.html]: Should this be inside your <html> tags?

**Comment by nickcolley** [comment]
Woop, this is looking great, there's just one thing I noticed.

The boilerplate
```html
<html></html><body></body>
```

Is repeated a few times (html should also have a lang attribute)

It'd be great if we could have a base template that the other templates extend from so you don't have to repeat those.

Only can review the code at the moment, will try and build and do a proper review later if I get the chance!

**Comment by brad-j** [comment]
@nickcolley Should I put the opening <html lang="en"> and <body> in the head layout with the closing tags in the footer layout?

**Comment by nickcolley** [comment]
You can nest layouts more than once in Jekyll so you could do

base layout -> blog layout -> post

Here's an example of nesting on PouchDB

https://github.com/pouchdb/pouchdb/blob/master/docs/_layouts/default.html

https://github.com/pouchdb/pouchdb/blob/master/docs/_layouts/post.html

https://github.com/pouchdb/pouchdb/blob/master/docs/_posts/2014-04-01-pouchdb-2.1.0.md

default -> post -> actual post

You can think of this like each layout is a component that accepts other components rather than wrapping a view with two components.

I hope I made sense, sorry for not explaining well in the first place!

**Comment by brad-j** [comment]
I think I get it. I'll get to it when I get home tonight. Thanks for the help @nickcolley!

**Comment by Charlotteis** [comment]
Hey @brad-j were you interested in continuing on with this? Hope you're well! :sparkles:

**Comment by Charlotteis** [comment]
Closing this as the work was done elsewhere. Thanks so much for your work, though!

**Pull request closed without merging**